install_notifier.js

```diff
@@ -0,0 +1,7 @@
+// Dispatch a message to every URL that's in the manifest to say that RECAP is
+// installed.
```

Comment:
Perhaps a little more clarity of purpose:

```patch
diff --git a/install_notifier.js b/install_notifier.js
index 2bac7dd..e2225e3 100644
--- a/install_notifier.js
+++ b/install_notifier.js
@@ -1,2 +1,4 @@
-// Dispatch a message to every URL that's in the manifest to say that RECAP is
-// installed.
+// Dispatch a message to every URL in the manifest for this file
+// to provide the RECAP version. This allows webpages to take
+// action based on the presence of the extension and its version.
+// This is only allowed for a small whitelist of domains.
 ```

Reply:
Sure. Tweaked.